### DISCUSSION TITLE
#19. VATZ biweekly meeting at 2022-10-12 08:00:00 -06:00:00

### DISCUSSION THREAD

**Discussion by xellos00** [dsrvlabs/vatz#254] [General]
### 1. Overall
<img width="1210" alt="image" src="https://user-images.githubusercontent.com/6308023/192920045-32c3f217-2c64-4800-8050-a3760eb63f90.png">

### 2. Statistic Rate
Sprint | Issue fulfillment | progress rate(%)
--: | :--: | :--:
17|  2/19 | 10
18|  1/18 | 5

---

- Finished
  - #222
    
- Move to Sprint 19
   - #246
   - #242 
   - #238
   - #226  
   - #202 
   - #197 
   - #187 
   - #175 
   - #174 
   - #146 
   - #122 
   - #121 
   - #119 
   - #87
   - #85
     
---

When you raise PR: 
   - Make a single test case at least if there's any update
   - Go Test SC
   - Go lint 
   - Go Build
   
---

### Discussion
- Github action 
  - Go Test 
  - Go Lint - ok!!
  - Go Build - ok!!

- Test
   - Unit Test
      - for PR, make a single test case at least if there's any update. 
         - attached SC  (temporarily til repo to be public) 
             - Build & Start  
             - coverage
      - Separate milestone:     
   - Integration test. (@xellos00 make another issue to discuss)
   
---

- Node Watcher : https://github.com/dsrvlabs/vatz/issues/273

**Comment by xellos00** [comment]
# 동요

## Developments

- https://github.com/dsrvlabs/vatz/issues/248
   - https://github.com/dsrvlabs/vatz/issues/87
   - https://github.com/dsrvlabs/vatz/issues/228 
- https://github.com/dsrvlabs/vatz/issues/240
   - https://github.com/dsrvlabs/vatz/issues/174
   - https://github.com/dsrvlabs/vatz/issues/246
   - https://github.com/dsrvlabs/vatz/issues/261
 
## Bug fix
- https://github.com/dsrvlabs/vatz/issues/259
- https://github.com/dsrvlabs/vatz/issues/262
- https://github.com/dsrvlabs/vatz/issues/180

**Comment by rootwarp** [comment]
# 준교
- https://github.com/dsrvlabs/vatz/issues/256
  - https://github.com/dsrvlabs/vatz/pull/260

**Reply by xellos00** [reply]
@rootwarp  
Please, Add md file for CLI and RPC  with new issue later.

**Comment by kim201212** [comment]
# 형기
- https://github.com/dsrvlabs/vatz/issues/236
   - https://github.com/dsrvlabs/vatz/issues/243
   - https://github.com/dsrvlabs/vatz/issues/266
   - https://github.com/dsrvlabs/vatz/issues/268
- https://github.com/dsrvlabs/vatz/issues/271

**Comment by skonhwang** [comment]
# 승곤
- https://github.com/dsrvlabs/vatz/issues/267
    - https://github.com/dsrvlabs/vatz-plugin-cosmoshub/pull/16

**Comment by gnongs** [comment]
# 진홍
## Developments
- https://github.com/dsrvlabs/vatz/issues/272
- #236 
  - #269 

## Bug fix  
- #262

**Comment by meetrick** [comment]
# 황제
## Developments
- #272 
- #236 
- #183

**Comment by heejin-github** [comment]
# 희진
- #236 
  - #265 
  - https://github.com/dsrvlabs/vatz-plugin-cosmoshub/pull/13
  - https://github.com/dsrvlabs/vatz-plugin-cosmoshub/pull/14
  - https://github.com/dsrvlabs/vatz-plugin-cosmoshub/pull/15

**Comment by xellos00** [comment]
@qkrybin 
Create an issue to record items, if there's anything we need on VATZ as first open-source user.
